frontend_lib/src/component/DropdownMenu/DropdownMenu.styl

```diff
@@ -31,6 +31,9 @@
     margin-right textSpacing
   &:hover:not(:active)
     background-color grey-hover
+  & > a > div
+    height 100%
+    width 100%
```

Comment:
try to surround the <Link> with the div containing the data-toggle and data-target please
it could avoid having to rely on such css rules

If it doesn't work or if it creates more complicated css rules, it's ok to keep it way you first implemented it

Reply:
same problem but the other way around (the link is only around the text and a rule is needed to make it occupy the whole button), I added a class to make the rule more specific

Reply:
ok
that's good